src/js/components/locations-configuration/AddBinModal.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+
+import PropTypes from 'prop-types';
+import { getTranslate } from 'react-localize-redux';
+import { connect } from 'react-redux';
+import Alert from 'react-s-alert';
+
+import { hideSpinner, showSpinner } from 'actions';
+import ModalWrapper from 'components/form-elements/ModalWrapper';
+import apiClient from 'utils/apiClient';
+import Translate, { translateWithDefaultMessage } from 'utils/Translate';
+
+
+class AddBinModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      values: {
+        active: true,
+      },
+    };
+  }
+
+
+  handleSubmit(values) {
+    this.props.showSpinner();
+    apiClient.post('/openboxes/api/locations/', {
```

Comment:
Let's rename `locType` into `locationType` and as a request body pass: `flattenRequest({ ...values, parentLocation: { id: this.props.locationId } })` (flattenRequest is a util funciton you can find in the apiClient)